minitwit/src/main/java/WebApplication.java

```diff
@@ -84,6 +89,44 @@ public static String render(Map<String, Object> model, String templatePath) {
         }
     }
 
+    public static Route serveFollowPage = (Request request, Response response) -> {
+
+        var db = new SQLite();
+        var conn = db.getConnection();
+
+        var insert = conn.prepareStatement("insert into follower (\n" +
+                "                who_id, whom_id) values (?, ?)");
+
+        var whom_id = getUserID(db, request.params(":username"));
+
+        insert.setInt(1, 999); //TODO: Get current users_id
```

Comment:
I think we can use the session attributes here no?

Reply:
I havent really looked into the session attributes thing yet, thought it werent implemented since we couldnt login yet